src/validators/unixdatetime.ts

```diff
@@ -0,0 +1,163 @@
+import {
+  CodeGenResult,
+  isValidType,
+  ValidatorBase,
+  ValidatorBaseOptions,
+  ValidatorExportOptions,
+  ValidatorOptions
+} from '../common'
+import { NotIntegerFail, NotStringFail, OutOfRangeFail, RequiredFail, ValidationFailure } from '../errors'
+import { validateString } from './string'
+
+export function isUnixDateTime(value: unknown, context?: string): value is string {
+  const errors = validateUnixDateTime(value, context)
+  if (errors.length === 0) {
+    return true
+  }
+  return false
+}
+
+export function validateUnixDateTime(value: unknown, context?: string): ValidationFailure[] {
+  const stringError = validateString(value, 1, 20, context)
+  if (!isValidType<string>(value, stringError)) {
+    return stringError
+  }
+
+  const numValue = Number(value)
```

Comment:
That is not a good way to convert a string to a number. please read the docs here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number
I would suggest using this instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt